webdriver-spec.html

```diff
@@ -1259,17 +1259,11 @@
 
  <tr>
   <td>POST</td>
-  <td>/session/{<var>session id</var>}/execute/sync</td>
+  <td>/session/{<var>session id</var>}/execute</td>
   <td><a>Execute Script</a></td>
  </tr>
 
  <tr>
-  <td>POST</td>
```

Comment:
Async execute should remain. The difference between `execute` and `execute async` is that the script for `execute async` would get access to the resolve function, as per @jleyba's suggestion here: https://github.com/w3c/webdriver/issues/381#issuecomment-276274426